scripts/check.unused.mjs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+import { readFileSync } from 'node:fs';
+import { basename, dirname, resolve } from 'node:path';
+import { findFiles } from './utils.mjs';
+
+const RED = '\x1b[31m';
+const GREEN = '\x1b[32m';
+const NC = '\x1b[0m'; // No Colour
+
+const DATA_DIR = 'src/frontend/src';
+const DATA_DIR_PATH = resolve(process.cwd(), DATA_DIR);
+
+const findSvelteFiles = (dir) => findFiles({ dir, extensions: ['.svelte'] });
+
+const main = async () => {
+	console.log(`${NC}Scanning ${DATA_DIR} folder to find all .svelte files`);
+
+	const allSvelteFiles = findSvelteFiles(DATA_DIR_PATH);
+
+	const potentialUnusedFiles = allSvelteFiles.filter((file) => !dirname(file).includes('routes'));
+
+	const unusedFiles = allSvelteFiles.reduce((acc, file) => {
```

Comment:
Yes, you are right: I was trying to leave it more functional, but this way it can be faster indeed.

However, I still leave the check in the end too, since it should consider the improbable situation where we arrive there with empty `unusedFiles`: for example, if `allSvelteFiles` is empty (again, it is very improbable, but for consistency...)